ProvisionSample/Program.cs

```diff
@@ -295,6 +329,40 @@ static void Main(string[] args)
             }
         }
 
+        static async Task<Import> GetStatus(string workspaceCollectionName, string workspaceId, string importId)
+        {
+            using (var client = await CreateClient())
+            {
+                Import import;
+                try
+                {
+                    import = await client.Imports.GetImportByIdAsync(workspaceCollectionName, workspaceId, importId);
+                }
+                catch (Exception ex)
+                {
+                    // TODO Get HTTP response body?
+                    Console.ForegroundColor = ConsoleColor.Red;
+                    Console.WriteLine("Exception: {0}", ex);
+                    return null;
+                }
+
+                Console.WriteLine("Name:     {0}", import.Name);
```

Comment:
I don't have a strong opinion here, but my preference would be that the Get* method just returns the Import and defers writing output to the caller.

The new case 0 for getting the status of an import is writing the ImportState, since that is its concern. A user who selects case 0 might be surprised to see so much more output. 

We could add a case 'a' to do a debug dump of the full import object. Alternatively, if this was meant for debugging purposes, we could either remove this output or use Debug.WriteLine instead.